.github/workflows/pr-linter.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         config:
-          - { python-version: 3.8, tox-env: lint}
+          - { python-version: 3.10, tox-env: lint}
```

Comment:
Making `3.10` a `str` object, `"3.10"` may fix this PR.

Reply:
Ahh, nice catch. Feel free to make the change and push it if you want.

Reply:
Thank you, @ahopkins